src/views/monitoring/alert-manager/modules/alert-detail/AlertDetailDetails.vue

```diff
@@ -74,7 +69,7 @@ export default {
                 { name: 'acknowledged_at', label: i18n.t('MONITORING.ALERT.DETAIL.INFO.ACKNOWLEDGED'), disableCopy: true },
                 { name: 'resolved_at', label: i18n.t('MONITORING.ALERT.DETAIL.INFO.RESOLVED'), disableCopy: true },
             ]),
-            data: props.alertData || {},
+            data: computed(() => store.state.service.alert.alertData) || {},
```

Comment:
이 부분은 `computed(() => store.state.service.alert.alertData)`가 항상 참이라서, `|| {}` 를 만날 일이 없으니 아래처럼 바꾸면 좋을 것 같아요!

```suggestion
            data: computed(() => store.state.service.alert.alertData),
```

만약, `store.state.service.alert.alertData` 가 `null` or `undefined` 인 경우에는 data 에 writable한 일반 객체가 주어져야 하는 것이라면, 아래처럼 바꾸면 좋을 것 같습니다.

```suggestion
            data: store.state.service.alert.alertData ? computed(() => store.state.service.alert.alertData) : {},
```

혹은, `store.state.service.alert.alertData` 를 계속 감시하다가, `null` or  `undefined` 인 경우에는 빈 객체를 리턴해야 하는 경우라면 아래처럼 하면 좋을 것 같아요.ㅎㅎ

```suggestion
            data: computed(() => store.state.service.alert.alertData ?? {}),
```

Reply:
맨 아래꺼로 해놓을게요!